models/urbanczik_synapse.h

```diff
@@ -40,51 +40,66 @@
 namespace nest
 {
 
-/** @BeginDocumentation
-@ingroup Synapses
-@ingroup stdp
+/* BeginUserDocs: synapse, spike-timing-dependent plasticity
 
-Name: urbanczik_synapse - Synapse type for a plastic synapse after Urbanczik and Senn.
+Short description
++++++++++++++++++
 
-Description:
+Synapse type for a plastic synapse after Urbanczik and Senn
 
-urbanczik_synapse is a connector to create Urbanczik synapses as defined in [1] that can connect suitable
-multicompartment models. In contrast to usual STDP, the change of the synaptic weight does not only depend on the pre-
-and postsynaptic spike timing but also on the postsynaptic dendritic potential.
+Description
++++++++++++
 
-Urbanczik synapses require archiving of continuous quantities. Therefore they can only be connected to neuron models
-that are capable of doing this archiving. So far, the only compatible model is pp_cond_exp_mc_urbanczik.
+urbanczik_synapse is a connector to create Urbanczik synapses as defined in
+[1]_ that can connect suitable :ref:`multicompartment models
+<multicompartment-models>`. In contrast to usual STDP, the change of the
+synaptic weight does not only depend on the pre- and postsynaptic spike timing
```

Comment:
```suggestion
synaptic weight depends on the postsynaptic dendritic potential
```

Reply:
@jessica-mitchell What we wanted to express here is the following: Ordinary STDP depends on the pre- and postsynaptic spike timing. Urbanczik-Senn plasticity additionally depends on the dendritic membrane potential of the postsynaptic neuron.

Reply:
hmm I seem to have messed up my edit, but I understand what you mean - I was trying to remove the 'not' in the phrase and say " . . . synaptic weight depends on the postsynaptic dendritic potential, in addition to the pre- and postsynaptic spike timing". But upon thinking of this further I think the way you had it might be better in this case.

Reply:
@jessica-mitchell I see, but I like your suggestion, so let's replace the sentence with what you wrote.